sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -259,6 +259,43 @@ def test_table_schema_retrieve_with_direct_read(self):
           ]))
 
 
+class ReadUsingReadGbqTests(BigQueryReadIntegrationTests):
+  @pytest.mark.it_postcommit
+  def test_ReadGbq(self):
+    with beam.Pipeline(argv=self.args) as p:
+      expected_df = (
+          p | apache_beam.io.gcp.bigquery.ReadFromBigQuery(
+              method=beam.io.ReadFromBigQuery.Method.EXPORT,
+              table="apache-beam-testing:"
+              "beam_bigquery_io_test."
+              "dfsqltable_3c7d6fd5_16e0460dfd0",
+              output_type='BEAM_ROW'))
+    with beam.Pipeline(argv=self.args) as p:
+      actual_df = p | apache_beam.dataframe.io.read_gbq(
+          table="apache-beam-testing:"
+          "beam_bigquery_io_test."
+          "dfsqltable_3c7d6fd5_16e0460dfd0")
+    assert_that(expected_df, equal_to(actual_df))
```

Comment:
It's a bit odd to have two pipelines like this and verify them against each other. I also wouldn't expect this to work with a deferred dataframe instance. Is it actually working?

I think it would be preferable to just have an explicit expected value, as in the other ITs. Can we do that instead?